templates/R53-hostedzone.yaml

```diff
@@ -1,18 +1,21 @@
 Description: Setup a hosted zone for DNS
 AWSTemplateFormatVersion: 2010-09-09
 Parameters:
-  Department:
-    Description: The department for this resource (i.e. Computational Oncology)
-    Type: String
-  Project:
-    Description: The name of the project that this resource is used for (i.e. Resilience)
+  CostCenter:
```

Comment:
I don't think we need this if we just pass in a CostCenter cloudformation tag on deployment which i think is the preferred approach.

Reply:
I don't understand what you mean by passing in a tag on deployment, without tagging the resource itself we would have to rely on the account tag for the containing account (which is fine for now, but I would prefer to have the option to set a CostCenter tag on all resources that support tagging)

Reply:
`R53-hostedzone.yaml` is a template.  when deploying the template (via cloudformation) you can [pass tags to cloudformation](https://docs.aws.amazon.com/cli/latest/reference/cloudformation/deploy/index.html) which will recursively apply those tags to all resource in a template.  When deploying with [org-formation update-stacks](https://github.com/org-formation/org-formation-cli/blob/master/docs/task-files.md#update-stacks) setting the tags will pass those tags to cloudformation.

Reply:
oh, I forgot that org-formation can propagate those tags; yeah let's rely on that instead of passing the values into every template